core/storage/classifier/gae_models.py

```diff
@@ -12,28 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Models for storing the classification models."""
+"""Models for storing the classification data models."""
 
 from core.platform import models
-import utils
+
+import feconf
 
 from google.appengine.ext import ndb
 
 (base_models,) = models.Registry.import_models([models.NAMES.base_model])
 
 # Available choices of algorithms for classification.
-ALGORITHM_CHOICES = [
-    'LDAStringClassifier'
-]
+ALGORITHM_CHOICES = [classifier_details['algorithm_id'] for (
+    classifier_details) in feconf.INTERACTION_CLASSIFIER_MAPPING.values()]
 
 
-class ClassifierModel(base_models.BaseModel):
+class ClassifierDataModel(base_models.BaseModel):
     """Storage model for classifier used for answer classification.
 
-    The id of instances of this class has the form
+    The id of instances of this class is the job_request_id of the corresponding
+    TrainClassifierJobModel and has the form
```

Comment:
I think we would have to update this to ClassifierTrainingJobModel as per Sean's latest suggestion on my PR (even I thought ClassifierTrainingJobModel would be a better name).